config/default.js

```diff
@@ -240,6 +240,17 @@ module.exports = {
   },
   rtlLangs: ['ar', 'fa', 'he', 'ur'],
   defaultLang: 'en-US',
+  // Mapping for missing moment locales that we can still map to existing ones.
```

Comment:
would this be a little clearer ~ Some missing moment locales can be mapped to existing ones.